src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -57,6 +57,22 @@ def __init__(
         self._seen_tables: dict[str, str] = {}
         self._principal_grants = principal_grants
 
+    def get_remaining_tables(self) -> list[Table]:
+        table_rows: list[Table] = []
```

Comment:
Type hint is not required:

```suggestion
        table_rows = []
```

Reply:
Done